subsys/bluetooth/host/gatt.c

```diff
@@ -4684,6 +4698,13 @@ int bt_gatt_subscribe(struct bt_conn *conn,
 		}
 	}
 
+	/* if application didn't set required minimum seucrity for notification
+	 * use sec level from time of subscription
+	 */
+	if (params->min_security < BT_SECURITY_L1) {
+		params->min_security = conn->sec_level;
```

Comment:
Is the `min_security` stored in the settings for persistent use?

Reply:
yes, since it is now part of subsciption (eg to be reused with bt_gatt_resubscribe())